boards/pinetime/include/periph_conf.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2019 Inria
+ *               2019 Freie Universität Berlin
+ *               2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_pinetime
+ * @{
+ *
+ * @file
+ * @brief       Peripheral configuration for the PineTime
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+#include "cfg_clock_32_1.h"
+#include "cfg_i2c_default.h"
+#include "cfg_rtt_default.h"
+#include "cfg_spi_default.h"
+#include "cfg_timer_default.h"
+
```

Comment:
Should the battery ADC line be included here?

Reply:
folo :)